private/bufpkg/bufimage/bufimagebuild/builder.go

```diff
@@ -268,6 +272,32 @@ func (b *builder) warnInvalidImports(
 	return nil
 }
 
+var (
+	wktFiles map[string]struct{} // populated in isWKTFile func
+	wktOnce  sync.Once
+)
+
+func isWKTFile(filepath string) bool {
+	wktOnce.Do(func() {
+		wktFiles = map[string]struct{}{
```

Comment:
We actually have this data already, it's just unexported in `datawkt`. I wonder if we could change `wkt-go-data` to create a new exported `FilePaths []string` variable that contains this.

Reply:
https://github.com/bufbuild/buf/blob/main/private/gen/data/datawkt/datawkt.gen.go#L158

Reply:
Actually yeah, that's basically exactly this function.

Reply:
it in fact IS exactly this function :-)